consul/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
```

Comment:
Why ? I guess it's because I'm not familiar with the integration SDK, but why this line ? It's a guarantee, and we say we might break the backward compatibility with agent 6 ?